code/game/turfs/turf.dm

```diff
@@ -363,8 +365,6 @@ GLOBAL_LIST_EMPTY(station_turfs)
 	var/canPassSelf = CanPass(mover, get_dir(src, mover))
 	if(canPassSelf || (mover.movement_type & PHASING))
 		for(var/atom/movable/thing as anything in contents)
-			if(QDELETED(mover))
-				return FALSE //We were deleted, do not attempt to proceed with movement.
```

Comment:
🤨

Reply:
oh this is slightly incorrect, the only impure things that can delete mover in this loop is the Bump() called iff mover is phasing or the Cross() call if some weirdo eventually creates an override that can delete mover from there